acceptance/lib/facter/acceptance/user_fact_utils.rb

```diff
@@ -117,6 +117,16 @@ def custom_fact_content(key='custom_fact', value='custom_value', *args)
   end
         EOM
       end
+
+      # Return the correct shell path for Unix system under test
+      #
+      def user_shell(agent)
```

Comment:
Because I copied the code from somewhere else. Although now that I look it better this way otherwise you might assume your getting shell, when you really getting one of the following: ksh93, dash, or bash depending on the system your on.